setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='api-integration',
-    version='5.0.0',
+    version='5.0.1',
```

Comment:
@wasifarbisoft please revert this version to 5.0.0 as we updated this version for juniper release. we will release everything in the same version. and do not create this release as we are currently using branch name in custom.txt

Reply:
@ihtram ,
Fixed !